.lycheeignore

```diff
@@ -0,0 +1,19 @@
+john.doe@mycompany.com
```

Comment:
Are this files related to us at all? Looks like they are coming from some example?